app/Http/Controllers/AppController.php

```diff
@@ -294,12 +301,13 @@ public function app_post(Request $request){
         $success_response['status_code'] = '1';
 
 
-        if(!$request->has('user_id')){
-            $error_response['status']='Error : user_id not received.';
+        if(!$request->has('rollNo')){
+            $error_response['status']='Error : rollNo not received.';
             return response($error_response);
         }
 
-        $app_user = AppUser::find($request->user_id);
+        $id = AppUser::where('rollNo','=',$request->rollNo)->value('id');
+        $app_user = AppUser::find($id);
```

Comment:
Recommended : use `$app_user = AppUser::where('rollNo',$request->rollNo)->first();` and remove the previous statement.

Reply:
done